include/alpaka/intrinsic/IntrinsicCpu.hpp

```diff
@@ -10,7 +10,7 @@
 
 #include <bitset>
 #include <climits>
-#if __has_include(<bit>)
+#if(__cplusplus >= 202002L) && __has_include(<bit>)
 #    include <bit>
 #endif
```

Comment:
This is a pessimization in general, because compilers can provide the `<bit>` header in c++2a mode already, and not bump `__cplusplus`.

This is also dangerous on MSVC, because `__cplusplus` in C++20 mode is still `199711L`, unless the `/Zc:__cplusplus` flag is specified. So any user not defining this will also not get the `<bit>` header.

The right thing to do here IMO is to silence the warning.

Reply:
> This is also dangerous on MSVC, because `__cplusplus` in C++20 mode is still `199711L`

Wow, that is amazingly stupid. Alright, I'll just silence the warning then.